x/feerefunder/keeper/keeper_test.go

```diff
@@ -254,9 +282,18 @@ func TestDistributeAcknowledgementFee(t *testing.T) {
 		),
 	}, ctx.EventManager().Events())
 
-	feeInfo, err := k.GetFeeInfo(ctx, packet)
+	feeInfo, found := k.GetFeeInfo(ctx, packet)
 	require.Nil(t, feeInfo)
-	require.ErrorContains(t, err, "no fee info")
+	require.False(t, found, "no fee info")
```

Comment:
same at the line 361
```suggestion
	require.False(t, found, "no fee is expected to be info")
```